### PR TITLE
*: update PD configuration

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@nolouch Please involve technical review(s).

**Comment by nolouch** [comment]
PTAL @HunDunDM @rleungx

**Review by HunDunDM — APPROVED** [review]

File: pd-configuration-file.md [hunk 1]
```diff
@@ -349,6 +349,12 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 参考[Placement Rules 使用文档](/configure-placement-rules.md)
 + 4.0 实验性特性
 
+### `flow-round-by-digit` <span class="version-mark">从 v5.1 版本开始引入</span>
+
++ 打开 `flow-round-by-digit`
++ 默认值：3
++ 用于控制心跳上报的 Region 流量的离散化处理，减少流量信息变化引起的统计信息更新。表示对流量信息的末尾几位的数字四舍五入，比如流量 100512 会归约到 100500。默认值为 3。该配置替换了 `trace-region-flow`。
```
Comment (TomShawn) [inline pd-configuration-file.md:356]: 这篇文档里没有 `trace-region-flow`，这里是否也不需提呢？

File: pd-control.md [hunk 2]
```diff
@@ -446,10 +446,20 @@ export PD_ADDR=http://127.0.0.1:2379 &&
 
 - `enable-debug-metrics` 用于开启 debug 的 metrics。当设置为 true 时，PD 会开启一些 metrics，比如 `balance-tolerant-size` 等。
 
-- `enable-placement-rules` 用于开启 placement rules。
+- `enable-placement-rules` 用于开启 placement rules, 5.0 以上的版本默认开启。
```
Comment (TomShawn) [inline pd-control.md:449]: 这里包括 v5.0 版本吗？
```suggestion
- `enable-placement-rules` 用于开启 placement rules，在 v5.0 以上的版本默认开启。
```

File: pd-control.md [hunk 3]
```diff
@@ -446,10 +446,20 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
 
 - `store-limit-mode` 用于控制 store 限速机制的模式。主要有两种模式：`auto` 和 `manual`。`auto` 模式下会根据 load 自动进行平衡调整（实验性功能）。
 
+- `flow-round-by-digit` 用于控制心跳上报的 Region 流量的离散化处理，减少流量信息变化引起的统计信息更新。表示对流量信息的末尾几位的数字四舍五入，比如流量 100512 会归约到 100500。默认值为 3。该配置替换了 `trace-region-flow`。
+
+    设置 flow-round-by-digit 为 4：
```
Comment (TomShawn) [inline pd-control.md:455]:
```suggestion
    示例：将 `flow-round-by-digit` 的值设为 `4`：
```

File: pd-configuration-file.md (hunk 1 above)
Comment (TomShawn) [inline pd-configuration-file.md:356]:
```suggestion
+ PD 会对流量信息的末尾数字进行四舍五入处理，减少 Region 流量信息变化引起的统计信息更新。该配置项用于指定对 Region 流量信息末尾几位数字进行四舍五入。例如流量 `100512` 会归约到 `100500`。默认值为 `3`。该配置替换了 `trace-region-flow`。
```

File: pd-control.md [hunk 4]
```diff
@@ -446,10 +446,20 @@ export PD_ADDR=http://127.0.0.1:2379 &&
 
 - `enable-debug-metrics` 用于开启 debug 的 metrics。当设置为 true 时，PD 会开启一些 metrics，比如 `balance-tolerant-size` 等。
 
-- `enable-placement-rules` 用于开启 placement rules。
+- `enable-placement-rules` 用于开启 placement rules, 5.0 以上的版本默认开启。
 
 - `store-limit-mode` 用于控制 store 限速机制的模式。主要有两种模式：`auto` 和 `manual`。`auto` 模式下会根据 load 自动进行平衡调整（实验性功能）。
 
+- `flow-round-by-digit` 用于控制心跳上报的 Region 流量的离散化处理，减少流量信息变化引起的统计信息更新。表示对流量信息的末尾几位的数字四舍五入，比如流量 100512 会归约到 100500。默认值为 3。该配置替换了 `trace-region-flow`。
```
Comment (TomShawn) [inline pd-control.md:453]:
```suggestion
-  PD 会对流量信息的末尾数字进行四舍五入处理，减少 Region 流量信息变化引起的统计信息更新。该配置项用于指定对 Region 流量信息的末尾进行四舍五入的位数。例如流量 `100512` 会归约到 `100500`。默认值为 `3`。该配置替换了 `trace-region-flow`。
```

File: pd-configuration-file.md (hunk 1 above)
Comment (nolouch) [reply pd-configuration-file.md:356]: 这个配置没有公开过，只有部分用户救急用。可以理解为隐藏参数，这里提了一下。

**Review by TomShawn — APPROVED** [review]
LGTM

File: pd-control.md [hunk 5]
```diff
@@ -446,10 +446,20 @@ export PD_ADDR=http://127.0.0.1:2379 &&
 
 - `enable-debug-metrics` 用于开启 debug 的 metrics。当设置为 true 时，PD 会开启一些 metrics，比如 `balance-tolerant-size` 等。
 
-- `enable-placement-rules` 用于开启 placement rules。
+- `enable-placement-rules` 用于开启 placement rules，在 v5.0 以上的版本默认开启。
 
 - `store-limit-mode` 用于控制 store 限速机制的模式。主要有两种模式：`auto` 和 `manual`。`auto` 模式下会根据 load 自动进行平衡调整（实验性功能）。
 
+-  PD 会对流量信息的末尾数字进行四舍五入处理，减少 Region 流量信息变化引起的统计信息更新。该配置项用于指定对 Region 流量信息的末尾进行四舍五入的位数。例如流量 `100512` 会归约到 `100500`。默认值为 `3`。该配置替换了 `trace-region-flow`。
```
Comment (TomShawn) [inline pd-control.md:453]:
```suggestion
- PD 会对流量信息的末尾数字进行四舍五入处理，减少 Region 流量信息变化引起的统计信息更新。该配置项用于指定对 Region 流量信息的末尾进行四舍五入的位数。例如流量 `100512` 会归约到 `100500`。默认值为 `3`。该配置替换了 `trace-region-flow`。
```

File: pd-configuration-file.md [hunk 6]
```diff
@@ -349,6 +349,12 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 参考[Placement Rules 使用文档](/configure-placement-rules.md)
 + 4.0 实验性特性
 
+### `flow-round-by-digit` <span class="version-mark">从 v5.1 版本开始引入</span>
+
++ 打开 `flow-round-by-digit`
```
Comment (TomShawn) [inline pd-configuration-file.md:354]: @nolouch 这一行看起来挺奇怪的，作为一个配置项的描述就是打开自己？

Comment (TomShawn) [reply pd-configuration-file.md:354]: 这一行应该去掉

File: pd-configuration-file.md [hunk 7]
```diff
@@ -349,6 +349,12 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
...
++ 默认值：3
++ PD 会对流量信息的末尾数字进行四舍五入处理，减少 Region 流量信息变化引起的统计信息更新。该配置项用于指定对 Region 流量信息的末尾进行四舍五入的位数。例如流量 `100512` 会归约到 `100500`。默认值为 `3`。该配置替换了 `trace-region-flow`。
```
Comment (TomShawn) [inline pd-configuration-file.md:356]: 100500 --> 101000

File: pd-configuration-file.md (hunk 6 above)
Comment (TomShawn) [reply pd-configuration-file.md:354]: https://github.com/pingcap/docs-cn/pull/6577

File: pd-configuration-file.md (hunk 7 above)
Comment (TomShawn) [reply pd-configuration-file.md:356]: https://github.com/pingcap/docs-cn/pull/6577

**Pull request merged**